<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtTestPylosRandomPlayDialog</class>
 <widget class="QDialog" name="QtTestPylosRandomPlayDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Random play</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ToolTestPylos.qrc">
    <normaloff>:/images/R.png</normaloff>:/images/R.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="group_game_type">
     <property name="title">
      <string>Game type</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="radio_basic">
        <property name="text">
         <string>Basic</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radio_advanced">
        <property name="text">
         <string>Advanced</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_n_games">
     <property name="text">
      <string>Number of games to play</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="edit_n_games">
     <property name="text">
      <string>100</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_start">
     <property name="text">
      <string>Play games</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_games_played">
     <property name="text">
      <string>Games played</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="bar_progress">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_percentage_won_player1">
     <property name="text">
      <string>Percentage of games won by player 1</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="bar_won_player1">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ToolTestPylos.qrc"/>
 </resources>
 <connections/>
</ui>
